grafana-plugin/package.json

```diff
@@ -127,7 +128,7 @@
     "webpack-livereload-plugin": "^3.0.2"
   },
   "engines": {
-    "node": "~18.16.0"
+    "node": ">18.16.0"
```

Comment:
🤣

Reply:
I put them in based on advice like this: https://stackoverflow.com/questions/46174085/should-changes-in-a-package-json-file-be-commited-to-a-repository-as-well

...but actually I have no experience working with node and yarn requirements files.  I thought it would help to have dependencies committed, but why do you think we shouldn't?